app/controllers/client_notifications_controller.rb

```diff
@@ -0,0 +1,56 @@
+class ClientNotificationsController < ApplicationController
+  before_action :set_client_notification, only: [:edit, :update, :destroy]
+  before_action :translate_model_name, only: [:update, :destroy]
```

Comment:
after the translate model name method is changed like explained below, you can drop this before_action